xgboost-predictor/src/main/java/biz/k11i/xgboost/Predictor.java

```diff
@@ -347,8 +367,17 @@ public String getObjName() {
         return name_obj;
     }
 
+    public ObjFunction getObjective() {
+        return obj;
+    }
+
     public float getBaseScore() {
         return base_score;
     }
 
+    // for testing/development only
+    void setBaseScore(float base_score) {
+        this.base_score = base_score;
+    }
+
```

Comment:
I know it a draft, just note for me.

Those methods are not used, is it used by the users of xgboost-predictor library?

Reply:
getObjective is used in the new PR, setBaseScore seems unused right now - probably intended for test but no longer needed, I will check

Reply:
There was a test missing, now setBaseScore is used by the newly added test.